internal/security/bootstrapper/command/setupacl/acltokens.go

```diff
@@ -0,0 +1,413 @@
+/*******************************************************************************
+ * Copyright 2021 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+
+package setupacl
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/bootstrap/startup"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients"
+)
+
+// AgentTokenType is the type of token to be set on the Consul agent
+type AgentTokenType string
+
+const (
+	/*
+	 * The following are available agent token types that can be used for setting the token to Consul's agent
+	 * For the details for each type, see reference https://www.consul.io/commands/acl/set-agent-token#token-types
+	 */
+	// DefaultType is agent token type "default" to be set
+	DefaultType AgentTokenType = "default"
+	// AgentType is agent token type "agent" to be set
+	AgentType AgentTokenType = "agent"
+	// MasterType is agent token type "master" to be set
+	MasterType AgentTokenType = "master"
+	// ReplicationType is agent token type "replication" to be set
+	ReplicationType AgentTokenType = "replication"
```

Comment:
Why do we need all of these defined if we only ever use AgentType in the code?

Reply:
don't need to just for the sake of completeness